QGL/ChannelLibraries.py

```diff
@@ -389,12 +389,16 @@ def new_source(self, label, model, address, power=-30.0, frequency=5.0e9, refere
         self.add_and_update_dict(thing)
         return thing
 
-    def set_control(self, qubit_or_edge, transmitter, generator=None):
+    def set_control(self, qubit_or_edge, transmitter, generator=None, aps1_chan=None):
         quads   = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalQuadratureChannel)]
         markers = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalMarkerChannel)]
 
         if isinstance(transmitter, Channels.Transmitter) and len(quads) > 1:
-            raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            #raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            print("Warning: In set_control the Transmitter must have a single quadrature channel if using APS2. Assuming APS1.")
+            if aps1_chan != '12' and aps1_chan != '34':
```

Comment:
How about redefining channel conventions like for APS2:
https://github.com/BBN-Q/QGL/commit/5fd7a903caec83ef615c2ca14f3e31573af7277c

Reply:
then it would be something like `if int(aps1_chan) > 2`: